src/jobservice/migration/migrator_v180.go

```diff
@@ -250,6 +250,11 @@ func getScoreByID(id string, conn redis.Conn, ns string) (int64, error) {
 
 // Get periodic policy object by the numeric ID
 func getPeriodicPolicy(numericID int64, conn redis.Conn, ns string) (*period.Policy, error) {
+	// close this inner connection here
+	defer func() {
+		_ = conn.Close()
```

Comment:
not very relevant to the fix, but shall we log the error of closing conn?

Reply:
No, the root cause is this conn is not closed.

Remove the maxConn in the pool creation is an enhanced fix 

for the error handling, I don't think we need because the reason I add _ is dismissing the best practice tips, no risk to the code.

Reply:
Thanks for fixing it anyway.